### PR TITLE
[D1] Add redirect, add `wrangler d1 execute` changelog and docs change

### PR DESCRIPTION
**Pull request by rozenmd**
Work in progress:
- [x] redirect d1/changelog to d1/platform/changelog
- [x] add changelog entry for `wrangler d1 execute`
- [x] update existing documentation for `wrangler d1 execute`
- [x] update existing documentation for `wrangler d1 migrations apply`
- [x] update docs for importing data -> Convert SQLite database files 

This PR is to be merged only once https://github.com/cloudflare/workers-sdk/pull/5203 is merged

### CODE REVIEW

File: content/_redirects [hunk 1]
```diff
@@ -214,6 +214,7 @@
 /constellation/ /workers-ai/ 301
 
 # D1
+/d1/changelog /d1/platform/changelog/ 301
```
Comment (rozenmd) [inline content/_redirects]: I noticed we have many links in the wild pointing at `d1/changelog`, which 404s

**Review by vy-ton — APPROVED** [review]

File: content/d1/build-databases/import-data.md [hunk 2]
```diff
@@ -92,19 +92,30 @@ For example, if you have a raw SQLite dump called `db_dump.sqlite3`, run the fol
...
 
 ## Troubleshooting
 
 If you receive an error when trying to import an existing schema and/or dataset into D1:
 
-* Ensure you are importing data in SQL format (typically with a `.sql` file extension). See [how to convert SQLite files](#convert-sqlite-database-files) if you have a `.sqlite3` database dump.
-* Make sure the schema is [SQLite3](https://www.sqlite.org/docs.html) compatible. You cannot import data from a MySQL or PostgreSQL database into D1, as the types and SQL syntax are not directly compatible.
-* If you have foreign key relationships between tables, ensure you are importing the tables in the right order. You can't refer to a table that doesn't yet exist.
-* If you get `"cannot start a transaction within a transaction"`, make sure you have removed `BEGIN TRANSACTION` and `COMMIT` from your dumped SQL statements.
+- Ensure you are importing data in SQL format (typically with a `.sql` file extension). See [how to convert SQLite files](#convert-sqlite-database-files) if you have a `.sqlite3` database dump.
```
Comment (Maddy-Cloudflare) [inline content/d1/build-databases/import-data.md]:
```suggestion
- Ensure you are importing data in SQL format (typically with a `.sql` file extension). Refer to [how to convert SQLite files](#convert-sqlite-database-files) if you have a `.sqlite3` database dump.
```

File: content/d1/build-databases/import-data.md [hunk 3]
```diff
@@ -92,19 +92,30 @@ For example, if you have a raw SQLite dump called `db_dump.sqlite3`, run the fol
...
+- Make sure the schema is [SQLite3](https://www.sqlite.org/docs.html) compatible. You cannot import data from a MySQL or PostgreSQL database into D1, as the types and SQL syntax are not directly compatible.
+- If you have foreign key relationships between tables, ensure you are importing the tables in the right order. You can't refer to a table that doesn't yet exist.
```
Comment (Maddy-Cloudflare) [inline content/d1/build-databases/import-data.md]:
```suggestion
- If you have foreign key relationships between tables, ensure you are importing the tables in the right order. You cannot refer to a table that does not yet exist.
```

File: content/d1/build-databases/import-data.md [hunk 4]
```diff
@@ -92,19 +92,30 @@ For example, if you have a raw SQLite dump called `db_dump.sqlite3`, run the fol
...
+- If you get `"cannot start a transaction within a transaction"`, make sure you have removed `BEGIN TRANSACTION` and `COMMIT` from your dumped SQL statements.
```
Comment (Maddy-Cloudflare) [inline content/d1/build-databases/import-data.md]:
```suggestion
- If you receive a `"cannot start a transaction within a transaction"` error, make sure you have removed `BEGIN TRANSACTION` and `COMMIT` from your dumped SQL statements.
```

File: content/d1/build-databases/import-data.md [hunk 5]
```diff
@@ -92,19 +92,30 @@ For example, if you have a raw SQLite dump called `db_dump.sqlite3`, run the fol
...
 
 ## Next Steps
 
-* Read the SQLite [`CREATE TABLE`](https://www.sqlite.org/lang_createtable.html) documentation
-* Learn how to [use the D1 client API](/d1/build-databases/query-databases/) from within a Worker
-* Understand how [database migrations work](/d1/reference/migrations/) with D1
+- Read the SQLite [`CREATE TABLE`](https://www.sqlite.org/lang_createtable.html) documentation
```
Comment (Maddy-Cloudflare) [inline content/d1/build-databases/import-data.md]:
```suggestion
- Read the SQLite [`CREATE TABLE`](https://www.sqlite.org/lang_createtable.html) documentation.
```

File: content/d1/build-databases/import-data.md [hunk 6]
```diff
@@ -92,19 +92,30 @@ For example, if you have a raw SQLite dump called `db_dump.sqlite3`, run the fol
...
+- Learn how to [use the D1 client API](/d1/build-databases/query-databases/) from within a Worker
```
Comment (Maddy-Cloudflare) [inline content/d1/build-databases/import-data.md]:
```suggestion
- Learn how to [use the D1 client API](/d1/build-databases/query-databases/) from within a Worker.
```

File: content/d1/build-databases/import-data.md [hunk 7]
```diff
@@ -92,19 +92,30 @@ For example, if you have a raw SQLite dump called `db_dump.sqlite3`, run the fol
...
+- Understand how [database migrations work](/d1/reference/migrations/) with D1
```
Comment (Maddy-Cloudflare) [inline content/d1/build-databases/import-data.md]:
```suggestion
- Understand how [database migrations work](/d1/reference/migrations/) with D1.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**